modular_skyrat/modules/vore/code/datums/components/vore.dm

```diff
@@ -0,0 +1,141 @@
+/datum/component/vore
```

Comment:
Adding to what i said, this component is poorly utilized IMHO. 
Please reconsider the whole belly object deal.

Reply:
vore component, vore_prefs datum and belly object are all non-negotiable and work to benefit it well, as well as for ease of use during coding and less overhead for the server